docs.typ

```diff
@@ -0,0 +1,387 @@
+#import "./docs-template.typ": *
+
+#import "@preview/tidy:0.1.0"
+
+#let version = toml("/typst.toml").package.version
+#let import-statement = "#import \"@preview/tidy:" + version + "\""
+
+#show: docs-template.with(
+  title: "The Notebookinator", subtitle: "Easy formatting for robotics notebooks.", abstract: [
+    Welcome to the Notebookinator, a Typst package meant to simply the notebooking
```

Comment:
**Simplify** instead of simply

Reply:
Did you ever apply this change?

Reply:
No I didn't know I could change it, I've never done a pull request before so I thought you would see this and change it, sorry.